### DISCUSSION TITLE
What is the least attractive color? 🪞😿

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#946] [Galaxy Trivia] [answered]
**Bonus:** Where can you see it being used? #RGB

**Comment by tomerf** [comment]
Brown #A52A2A 💩 🍫 
It's used in woods, boots and chocolate.

**Comment by peterbaccaro** [comment, marked as answer]
Pantone #746542
Described as a drab dark brown and informally dubbed the "ugliest colour in the world". It is the colour for plain tobacco and cigarette packaging.
